portal/views/patients.py

```diff
@@ -50,18 +50,20 @@ def patients_root():
         #todo iterate on users? here to add due_date stuff
         # FIXME kludge for random demo data
         for user in org.users:
-            user.due_date = datetime(random.randint(2016, 2017), random.randint(1, 12), random.randint(1, 28))
-            timedelta_days = user.due_date - datetime.today()
-            timedelta_days = timedelta_days.days
-            if timedelta_days < 0:
-                desc = 'overdue'
-            elif timedelta_days < 30:
-                desc = 'due'
-            else:
-                desc = 'not due'
-            user.random_due_date_status = desc
-            user.due_date = user.due_date.strftime('%d %b %Y')
-            user.due_date = user.due_date.lstrip('0') # remove any leading 0 from day
+            if not user.deleted:
```

Comment:
This check isn't necessary - what is necessary is to avoid sending any deleted users into the patient list (my oversight).  I'll submit this change to this branch shortly.